diff --git a/stratagems/sfo/general/lib_general.tpa b/stratagems/sfo/general/lib_general.tpa
index 289c2ca..119f28f 100644
--- a/stratagems/sfo/general/lib_general.tpa
+++ b/stratagems/sfo/general/lib_general.tpa
@@ -57,11 +57,11 @@ DEFINE_ACTION_FUNCTION ~compile_ssl_to_bcs_core~ INT_VAR skip_bcs=0 STR_VAR scri
                      REPLACE_TEXTUALLY "\" "/"
                 END
 		AT_NOW ~chmod 755 %MOD_FOLDER%/%ssl_loc%/ssl.pl~
-                AT_NOW ~/usr/bin/perl %MOD_FOLDER%/%ssl_loc%/ssl.pl %scsargs% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %workspace%/autolib %booleanstring%~ EXACT	  
+                AT_NOW ~%MOD_FOLDER%/%ssl_loc%/ssl.pl %scsargs% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %workspace%/autolib %booleanstring%~ EXACT	  
           END ELSE BEGIN
                 LAF check_ini STR_VAR ini=Use_Perl_Natively RET value END
                 ACTION_IF value BEGIN
-                   AT_NOW ~perl %MOD_FOLDER%\%ssl_loc%\ssl.pl %full_loc%\%script% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %workspace%\autolib %booleanstring%~
+                   AT_NOW ~%MOD_FOLDER%\%ssl_loc%\ssl.pl %full_loc%\%script% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %workspace%\autolib %booleanstring%~
 		END ELSE BEGIN
                    AT_NOW ~%MOD_FOLDER%\%ssl_loc%\ssl.exe %full_loc%\%script% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %workspace%\autolib %booleanstring%~
 		END
diff --git a/stratagems/sfo2e/lib_ssl.tph b/stratagems/sfo2e/lib_ssl.tph
index 642e2e4..3495112 100644
--- a/stratagems/sfo2e/lib_ssl.tph
+++ b/stratagems/sfo2e/lib_ssl.tph
@@ -95,7 +95,7 @@ BEGIN
 				REPLACE_TEXTUALLY EXACT_MATCH "\" "/"
 			END
 			AT_NOW ~chmod 755 %ssl_loc%/ssl.pl~
-			AT_NOW ~/usr/bin/perl %ssl_loc%/ssl.pl %scsargs% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %data_loc%/autolib %booleanstring%~ EXACT	  
+			AT_NOW ~%ssl_loc%/ssl.pl %scsargs% "%sslvariables%" -o %workspace%/ssl_out -l %ssllibrary% %data_loc%/autolib %booleanstring%~ EXACT	  
 		END
 	END
 END
